apps/server/src/modules/user-import/controller/api-test/import-user.api.spec.ts

```diff
@@ -528,7 +528,7 @@ describe('ImportUser Controller (API)', () => {
 							expect(listResponse.data.some((elem) => elem.userId === currentSchoolsUser.id)).toEqual(true);
 						});
 
-						it('should not respond with assigned users', async () => {
+						it.skip('should not respond with assigned users', async () => {
```

Comment:
Is this skip intended?

Reply:
yes, since the test does not work anymore due to the missing filtering, but the name of the endpoint suggests this behavior, so i did not want to delete it.